doc/api/util.md

```diff
@@ -858,6 +858,97 @@ added: v8.0.0
 * {symbol} that can be used to declare custom promisified variants of functions,
 see [Custom promisified functions][].
 
+## util.report.getReport([err])
```

Comment:
There should probably be a `## util.report` sub-heading?

Reply:
@addaleax - done, thnx.